CIME/SystemTests/mvk.py

```diff
@@ -9,35 +9,129 @@
 import os
 import json
 import logging
-
 from distutils import dir_util
 
-import CIME.test_status
-import CIME.utils
+from CIME import test_status
+from CIME import utils
 from CIME.SystemTests.system_tests_common import SystemTestsCommon
 from CIME.case.case_setup import case_setup
 from CIME.XML.machines import Machines
-
+from CIME.config import ConfigBase
+from CIME.SystemTests import test_mods
 
 import evv4esm  # pylint: disable=import-error
 from evv4esm.__main__ import main as evv  # pylint: disable=import-error
 
-evv_lib_dir = os.path.abspath(os.path.dirname(evv4esm.__file__))
+version = evv4esm.__version_info__
+
+assert version[0] <= 0 and version[1] <= 5, "Please install evv4esm less than 0.5.x"
```

Comment:
This would need updating each time there's a new EVV version, maybe this should be a lower bound instead, all the changes are compatible with evv4esm >= 0.5.0